import/translation_maps/format_map.properties

```diff
@@ -48,12 +53,14 @@ SoundRecording      = Audio
 TapeCartridge       = Software
 TapeCassette        = Software
 TapeReel            = Software
+Text                = Text
 Thesis              = Thesis
 Transparency        = Slide
 Unknown             = Unknown
 VideoCartridge      = Video
 VideoCassette       = VHS
 VideoDisc           = DVD
+VideoGame           = Video game
```

Comment:
I would capitalize "Game" here, since multi-word formats are fully capitalized elsewhere in the file.